x-pack/plugins/snapshot_restore/public/application/components/restore_snapshot_form/steps/step_logistics/system_indices_overwritten_callout.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import React, { FunctionComponent } from 'react';
+import { EuiCallOut, EuiLink } from '@elastic/eui';
+
+import { useCore } from '../../../../app_context';
+
+export const SystemIndicesOverwrittenCallOut: FunctionComponent = () => {
+  const { docLinks } = useCore();
+
+  return (
+    <EuiCallOut
```

Comment:
Thinking about accessibility, I wonder if screen-readers announce the content of this callout when the user enables the global state toggle? If not then I think we should consider rendering the callout even when the toggle is disabled. If the callout is too prominent to show at all times, maybe we could just present it as standard description text.

Reply:
Thanks for bringing this up! I've added an `aria-live` attribute so that it will be read by the screenreader when it is rendered.

> If not then I think we should consider rendering the callout even when the toggle is disabled. If the callout is too prominent to show at all times, maybe we could just present it as standard description text.

If there is a general concern that the callout is too prominent, I've also included an example of just using the `title` in the callout: https://github.com/elastic/kibana/pull/95104#discussion_r600104906. Adding it as a standard description text is a possibility as well.